src/rules/converters/file-name-casing.ts

```diff
@@ -0,0 +1,56 @@
+import { RuleConverter } from "../converter";
+
+const IGNORE_CASE_NOTICE = "ESLint (Unicorn plugin) does not support ignore as case";
+export const convertFileNameCasing: RuleConverter = tslintRule => {
+    return {
+        rules: [
+            {
+                ruleName: "unicorn/filename-case",
+                ...collectArguments(tslintRule.ruleArguments),
+            },
+        ],
+        plugins: ["unicorn"],
+    };
+};
+
+const collectArguments = (ruleArguments: any[]) => {
+    const notices: string[] = [];
+    const casesMap = new Map();
+    casesMap.set("camel-case", "camelCase");
+    casesMap.set("pascal-case", "pascalCase");
+    casesMap.set("kebab-case", "kebabCase");
+    casesMap.set("snake-case", "snakeCase");
```

Comment:
You can make a map with initialized properties in the constructor:

```ts
new Map([
    ["camel-case", "camelCase"],
    // ...
]);
```

That being said, I don't think this really needs to be a `Map`. Only `.get` is used on it, and always for keys guaranteed to exist. Can you switch it to just a regular object, and initialize it outside of `collectArguments` (since it's only assigned to and never modified)?